### PR TITLE
Apply stricter cors headers

### PR DESCRIPTION
**Pull request by SondreKanstad**
## Background
When developing authentication any host was allowed for easy development. For security reasons stricter CORS headers should be applied before merging and releasing in production.

## Solution
This was solved by removing anyHost and allowing a host set by an environment variable. In local development the env variable should be the same as the host of the frontend service `localhost:3000`.

### CODE REVIEW

**Review by JakobNH — APPROVED** [review]
Hva skal denne env-variablen være?

**Review by perzonas — APPROVED** [review]
Perhaps a different name, the environment variable has nothing to do with CORS, it is only used by CORS. This is just the frontend url

File: backend/src/main/kotlin/no/bekk/plugins/Cors.kt [hunk 1]
```diff
@@ -6,7 +6,7 @@ import io.ktor.server.plugins.cors.routing.*
 
 fun Application.configureCors() {
     install(CORS) {
-        anyHost() // Allow all origins, TODO: This should be more restrictive
+        allowHost(System.getenv("CORS_FRONTEND_URL_HOST"))
```
Comment (SondreKanstad) [inline backend/src/main/kotlin/no/bekk/plugins/Cors.kt:9]: Denne env variabelen skal være [hosten](https://developer.mozilla.org/en-US/docs/Web/API/URL/host) til frontenden. Så et bedre navn er nok `FRONTEND_URL_HOST`

Comment (SondreKanstad) [reply backend/src/main/kotlin/no/bekk/plugins/Cors.kt]: For localhost blir det `localhost:3000`

Comment (SondreKanstad) [reply backend/src/main/kotlin/no/bekk/plugins/Cors.kt]: Løst i commit [7addadc](https://github.com/kartverket/regelrett/pull/91/commits/7addadc07856ca5751e9325ec19a8336b19120cd)

**Pull request merged**